R/zzz.R

```diff
@@ -194,12 +196,14 @@ default_undesirable_functions <- all_undesirable_functions[names(all_undesirable
   "setwd",
   "sink",
   "source",
+  "structure",
   "Sys.setenv",
   "Sys.setlocale",
   "Sys.unsetenv",
   "trace",
   "undebug",
-  "untrace"
+  "untrace",
+  NULL
```

Comment:
it will shrink any future diff from appending to the end of the list (instead of 2 lines diff, one of which is just adding a trailing comma, the diff is focused on the one line added). Similar motivation in Python (c.f. `[1,2,]` and `(1,2,)`) and Go for allowing trailing `,` in constructors (in fact it's _required_ in Go: https://dave.cheney.net/2014/10/04/that-trailing-comma).

Also here:

https://github.com/r-lib/lintr/blob/8d420b5a035645116be2e1ad216ebea5cabe4ce5/R/shared_constants.R#L159

Basically accomplishing `rlang::list2()` for `c()`